static/src/stylesheets/module/journalism/_campaigns.scss

```diff
@@ -280,6 +287,35 @@
     }
 }
 
+/*---------- SUCCESS STATE TOGGLE -----------------*/
+
+.success.element-campaign .campaign--snippet {
+    .heading {
+        display: none;
+    }
+    button {
+        display: none;
+    }
+    form {
+        display: none;
+    }
+    div.success_box {
+        display: block;
+        color: $brightness-7;
+        padding: 8px 0px 10px 10px;
+        p {
+            font-weight: 700;
+            font-family: 'Guardian Text Sans Web', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif;
```

Comment:
can we reference the [typography variables](https://github.com/guardian/frontend/blob/8ae14c8e1ed4c41e3efe0bfb6f526dccab0bb237/static/src/stylesheets/pasteup/typography/_src.scss) here?